features/docs/writing_support_code/attachments.feature

```diff
@@ -51,4 +51,6 @@ Feature: Attachments
     
   Scenario: With json formatter, files can be attached given their path
     When I run `cucumber --format json features/attaching_screenshot_with_mediatype.feature`
-    Then the output should contain "embeddings\":[{\"mime_type\":\"image/png\",\"data\":\"Zm9v\"}]"
+    Then the output should contain "embeddings\":"
+    And the output should contain "\"mime_type\": \"image/png\","
+    And the output should contain "\"data\": \"Zm9v\""
```

Comment:
Is it possible to use a single step with the step definition `the output should contain:` with a multiline docstring?

Reply:
I'll try

Reply:
I've just tried. Actually because of the pretty json it is better the way you did with 3 distinct steps.